sky/cli.py

```diff
@@ -70,6 +70,7 @@
 from sky.utils import ux_utils
 from sky.utils.cli_utils import status_utils
 from sky.usage import usage_lib
+import time
```

Comment:
nit: we use google styleguide which sorts by section. Standard libs are in the first section, i.e., after L35.

Reply:
Moved.